autoinstall/Windows/win_server/bios/Autounattend.xml

```diff
@@ -30,15 +30,16 @@
                     <InstallTo>
                         <DiskID>0</DiskID>
                         <PartitionID>2</PartitionID>
-                    </InstallTo>
-                </OSImage>
+		    </InstallTo>
+		    <WillShowUI>OnError</WillShowUI>
+	        </OSImage>
             </ImageInstall>
             <DiskConfiguration>
                 <Disk wcm:action="add">
                     <CreatePartitions>
                         <CreatePartition wcm:action="add">
                             <Order>1</Order>
-                            <Size>549</Size>
+                            <Size>100</Size>

```

Comment:
Shrink the size?

Reply:
Yeah, according to the size in the OS installed manually.